salt/modules/win_iis.py

```diff
@@ -557,23 +558,23 @@ def create_binding(site, hostheader='', ipaddress='*', port=80, protocol='http',
     current_bindings = list_bindings(site)
 
     if name in current_bindings:
-        log.debug('Binding already present: {0}'.format(name))
+        log.debug('Binding already present: %s', name)
         return True
 
     if sslflags:
         ps_cmd = ['New-WebBinding',
                   '-Name', "'{0}'".format(site),
                   '-HostHeader', "'{0}'".format(hostheader),
                   '-IpAddress', "'{0}'".format(ipaddress),
-                  '-Port', "'{0}'".format(str(port)),
+                  '-Port', "'{0}'".format(six.text_type(port)),
                   '-Protocol', "'{0}'".format(protocol),
                   '-SslFlags', '{0}'.format(sslflags)]
     else:
         ps_cmd = ['New-WebBinding',
                   '-Name', "'{0}'".format(site),
                   '-HostHeader', "'{0}'".format(hostheader),
                   '-IpAddress', "'{0}'".format(ipaddress),
-                  '-Port', "'{0}'".format(str(port)),
+                  '-Port', "'{0}'".format(six.text_type(port)),
```

Comment:
`six.text_type` is unnecessary here, `.format()` will run `unicode()` on `port` automagically.